### ISSUE TITLE
Method 1 toggle not selecting necessary elements

### ISSUE THREAD

**Issue by Lefky** [vinorodrigues/bootstrap-dark-5#79]
When implementing the toggle I get the following error
`Uncaught TypeError: Cannot read properties of null (reading 'dataset')`

The issue seems to occur here
```
    function updateUI(color_p, id = 'css') {
        toggle_btn.checked = color_p === 'dark';

        if (isCssInit) {
            const el = document.querySelector('#' + id);
```
updateUI is called 2 times but never with id's passed. Css gets filled in instead but of course that will never match anything or at least not like it should.
Do you have some documentation on this? Or the toggling as a whole?
Cause I really would like to make it work.

PS: I expect the same issue for initColorCSS()

(JS padawan here)

**Comment by vinorodrigues** [comment]
Did you use the example?

https://vinorodrigues.github.io/bootstrap-dark-5/examples/cheatsheet-night.html

**Comment by Lefky** [comment]
Yes I exported the script block handling that to a javascript file and imported that on my site.

**Comment by Lefky** [comment]
Well if you put it like that...
I integrated it with my code and that doesn't work no. So I probably did something wrong, hence my question for extra info :)

**Comment by vinorodrigues** [comment]
Not sure how to help without looking at your code.  Kind of a conundrum for me though as I don't have the time to commit to support (and say as much both in the readme and in the issue submission template).

I would suggest reading the article I wrote for the [Bootstrap 4 variant](https://github.com/vinorodrigues/bootstrap-dark) of this project.  Also read the links I wrote in that article (especially the stuff from Thomas Steiner.  The toggle code I derived from [Radek's work](https://radek.io/posts/secret-darkmode-toggle/) - have a look at that to see if it helps.

(I'll close this as an issue as it's not an issue with the code I wrote - but you're welcome to continue writing on it.)
